examples/stable_diffusion/README.md

```diff
@@ -187,3 +187,35 @@ Inference will loop until the generated image. The result will be saved as `resu
 Run `python stable_diffusion.py --help` for additional options. A few particularly relevant ones:
 - `--image_path <str>`: the input image path for image to image inference.
 - `--img_to_img_example`: image to image example. The default input image is `assets/dog.png`, the default prompt is `amazing watercolor painting`.
+
+## Stable Diffusion Quantization encoded in QDQ format
+
+### Generate data for static quantization
+
+To get better result, we need to generate real data from original model instead of using random data for static quantization.
+
+First generate onnx unoptimized model:
+
+`python stable_diffusion.py --model_id stabilityai/sd-turbo --provider qdq --optimize --only_conversion`
```

Comment:
The `provider` definition in this script means which hardware backend to use. Can we use a different argument like `--format` for qdq?

Reply:
maybe I will rename it to cpu_qdq? as it is only a kind of optimization recipe

Reply:
Even for qdq, it can be run on all devices. Just your config's the default device is CPU. I would keep provider option for qdq and add `--format qdq` or `--qdq` for this recipe.

Reply:
use --provider cpu --format qdq